lib/log.js

```diff
@@ -61,6 +61,20 @@ Lug.prototype.event = function (name, data) {
   process.stdout.write(JSON.stringify(e) + '\n')
 }
 
+Lug.prototype.activityEvent = function(event, uid, request) {
+  var info = {
+    event: event,
+    uid: uid,
+  }
+  if ( request.headers['user-agent'] ) {
+    info.userAgent = request.headers['user-agent']
+  }
+  if ( request.payload && request.payload.service ) {
```

Comment:
nit: the space after the paren here is inconsistent with the style of the surrounding code